providers/aws/resources/aws.lr

```diff
@@ -2568,6 +2568,8 @@ private aws.ec2.image @defaults("arn") {
   ownerId string
   // Alias for the image owner
   ownerAlias string
+  // Date the image was created
+  creationDate time
```

Comment:
We don't have this totally standardized, but most of these are using the name `createdAt`. It would be good to make this that same name so we don't have more to deprecated in the future.

Reply:
updated!